spikes/stl_cxxsupport/main.cpp

```diff
@@ -94,5 +96,30 @@ auto main() -> int
 	auto back = span0.back();
 	log_info("span0.back() = %i", back);
 
+	log_info("Test std::range features");
+
+	constexpr auto range = [](const auto a, const auto b) {
+		auto x = std::ranges::views::iota(a) | std::ranges::views::take(b - a + 1);
+		return std::ranges::views::all(x);
+	};
+
+	for (auto x: range(1, 10)) {
+		log_info("x: %i", x);
+	}
+
+	for (auto y: range(20, 25)) {
+		log_info("y: %i", y);
+	}
+
+	log_info("Test utils::ranges::sequences features");
+
+	for (auto val: utils::ranges::sequence({.start = 0, .end = 5})) {
+		log_info("x: %i", val);
+	}
+
+	for (auto val: utils::ranges::sequence({.start = 10, .end = 20})) {
+		log_info("y: %i", val);
+	}
+
```

Comment:
Can add the other `sequence`

```suggestion
	for (auto val: utils::ranges::sequence({.start = 10, .end = 20})) {
		log_info("y: %i", val);
	}

	for (auto val: utils::ranges::sequence({.from = 101, .size = 10})) {
		log_info("z: %i", val);
	}

```